DataHub/Operation/01_ManageETLMappings.md

```diff
@@ -0,0 +1,314 @@
+  If the selected extension requires further configuration settings, the *Configuration* section is displayed below the *Settings* section.
+
+  > [Info] For detailed information about the data types, see [Data type list](../UserInterface/04_DataTypeList.md).   
+  For detailed information about the ETL extensions, see [ETL extensions list](../UserInterface/03_ETLExtensions.md).
+
+
+4. Select the appropriate tracking mode in the *Override change tracking mode* drop-down list. By default, the **Use default** option is preselected. The following options are available:
+  - **Use default**   
+    The mapping configured in the dependency of the specific entities is applied.
+  - **Automatic**   
+    The initial mapping is automatically applied. All value changes or reruns are applied automatically in the mapping.  
+  - **Semi-automatic**   
+    The initial mapping is automatically applied. Values changes or reruns are applied after confirmation.
+  - **Semi-automatic, changes must be confirmed by another user**   
+    The initial mapping as well as values changes or reruns are applied after confirmation by another user.
+  - **Manual**   
+    The initial mapping is automatically applied. Afterwards, the changes in the mapping must be triggered manually to be applied.
+
+
+5. Select the appropriate attribute that you want to assign to the destination attribute in the *Source attribute* drop-down list. All attributes with a data type that matches to the selected ETL extension are displayed in the drop-down list.
+
+6. If required, select or enter the corresponding configuration settings in the *Configuration* section.   
+
+7. Click the [SAVE] button in the upper right corner.   
+  The changes have been saved. The *Settings* section is hidden.
+
+  > [Info] Be aware that you have to [rerun the mapping](#rerun-an-etl-mapping) to apply the changes made to the attribute.
+
+8. Repeat step **2** to **7** for all attributes you want to map.
+
+
+
+### Add a custom mapping for a destination attribute
+
+When creating an ETL attribute set mapping, you define the language and the channel from which the values are taken for multi-language and multi-channel source attributes.
+Within an attribute set mapping, you can define custom channel and language attribute mappings for multi-language and multi-channel destination attributes.
+
+#### Prerequisites
+
+- At least one ETL attribute set mapping have been created, see [Create an ETL mapping](#create-an-ETL-mapping).
+- The destination attribute is multi-language and/or multi-channel.
+
+#### Procedure
+
+*DataHub > Settings > Tab ETL*
+
+![Attribute set mappings](../../Assets/Screenshots/DataHub/Settings/ETL/AttributeSetMappings.png "[Attribute set mappings]")
+
+1. Click the attribute set mapping you want to add a differing language or channel attribute mapping for a certain destination attribute.   
```

Comment:
```suggestion
1. Click the attribute set mapping you want to add a differing language or channel attribute mapping to for a certain destination attribute.   
```
Alternativ: Click the attribute set mapping where/to which you to add a different language or channel attribute mapping for a certain destination attribute.